python/hail/keytable.py

```diff
@@ -844,6 +844,19 @@ def persist(self, storage_level="MEMORY_AND_DISK"):
 
         return KeyTable(self.hc, self._jkt.persist(storage_level))
 
+    def unpersist(self):
+        """
+        Unpersists this keytable from memory/disk.
```

Comment:
we've been using "key table" or "table" instead of "keytable".

Here I'd use "table"